### DISCUSSION TITLE
Suggestions for future update

### DISCUSSION THREAD

**Discussion by face515** [Unlucky-Life/ankimon#79] [Ideas]
I realize it is probably too late for this update, but maybe next update. The pokemon that you are facing off against would act like a regular player, with the good, easy, hard, and again penalty. Based on the result you select, that changes the probability of the result the AI gets. This could help because you aren't as motivated to spam easy. After a match, there could be an option to decide whether to go to the medical center or go to another battle(could be an auto select mode in config). If you go to the center, you wait another turn to heal and then the next turn you go to battle.

**Comment by Unlucky-Life** [comment]
I really like that idea ! But yea i would inculde this in the next addon update.  But the question would be how would you calculate the chance for the easy, hard, good and again ? @face515

**Reply by face515** [reply]
@Unlucky-Life  A random function that's hard coded(maybe could increase the chance of doing the higher damage options as you level up) I don't know the exact percentages, but maybe 15,20,30,35? They could vary based on the result of your card and maybe time spent on card.

**Reply by Sabicool** [reply]
I'd like to stress that I feel it should be user controlled. I don't want to be pressured into picking good for the sake of not losing a battle but at the same time don't want it to be too easy. This granularity can only be achieved by the user. Ideally I would also like this to be configurable on a per deck preset because the type of cards and the overall difficulty of my cards varies from deck to deck.

**Reply by face515** [reply]
@Sabicool That makes sense! Another thing to consider is that when your deck has all new cards, you would by default get most wrong. This would make it a series of losing battles until the cards are memorized. In the options tab, something like that should be included. There could also be a difficulty slider. Maybe also a test option where it pulls up a fake deck. This deck could have options and you could see what always happens.

**Reply by Unlucky-Life** [reply]
@Sabicool @face515 Ok maybe i didnt understand the logic behind that ? I could think about making a pure xp gain for every card round or that you simply do damage for every card you do

**Reply by Sabicool** [reply]
@Unlucky-Life I was just suggesting that the chances for penalties based on user card rating (again, hard, good or easy) should be customisable on a per deck basis. I like the idea of there being the potential to have damage every round.
